x-pack/plugins/lens/public/datatable_visualization/_visualization.scss

```diff
@@ -1,3 +1,14 @@
 .lnsDataTable {
   align-self: flex-start;
 }
+
+.lnsDataTable__filter {
+  opacity: 0;
+  transition: opacity 250ms ease-in-out;
```

Comment:
The other tables don't seem to have a transition on the tooltip, why add this?

Reply:
Again, styles are aligned with inspector.

Reply:
```suggestion
  transition: opacity $euiAnimSpeedNormal ease-in-out;
```